src/main/scala/com/bradbrok/filmomatic/Direction.scala

```diff
@@ -0,0 +1,6 @@
+package com.bradbrok.filmomatic
+
+object Direction extends Enumeration {
+  type Direction = Value
+  val In, Out = Value
```

Comment:
Breaking it up even further sounds like a good idea. Direction, Bath, and State all are perfect.